internal/dbmodel/sql/postgres/1_16.sql

```diff
@@ -0,0 +1,5 @@
+-- 1_16.sql is a migration to delete identitymodel defaults
+DROP TABLE identity_model_defaults;
```

Comment:
it ok for this PR, but (and i'll keep complaining about this) we will need to be more careful in the future as there will be JIMMs running in the wild and at some point (hard as it might be to believe) some update will fail and customers will need to roll back the update..

so i propose we start dropping tables/columns a release after the code change - a more elaborate migration strategy needs to be discussed, of course

Reply:
as soon as kian is back we could have a meeting

Reply:
Well I think in future you'd have an up/down step in migrations, this will solve that. Once we're on golang-migrate, we can write a bunch of up/down @alesstimec so don't worry too much. For example, an up (which drops a table) could simply just move data into a jsonb table for situations like this.